awx/ui/client/src/inventories-hosts/inventories/related/hosts/edit/host-edit.controller.js

```diff
@@ -26,6 +26,7 @@
  				description: $scope.description,
  				enabled: $scope.host.enabled
  			};
+                        if (typeof $scope.host.instance_id !== 'undefined') host.instance_id = $scope.host.instance_id;
```

Comment:
@fosterseth This means that we'll sometimes reflect a zero-length string `""` to the api as the `instance_Id` in the `PUT` request Is this ok?

Reply:
![Screenshot from 2020-07-09 11-24-19](https://user-images.githubusercontent.com/9753817/87059080-e0fc1380-c1d6-11ea-9b2c-2dcd2d157d94.png)